src/mruby-zest/example/TextLine.qml

```diff
@@ -72,17 +72,17 @@ Widget {
         pos = @edit.pos if @edit
         ll = self.label
 
-        cursor_row = @edit.calc_cursor_y
+        cursorrow = @edit.cursor_row
```

Comment:
`cursor_row` instead of `cursorrow`
- it's the same as in `@edit.cursor_row`, which reduces the mental load (we don't have to think whether to spell it one way or another)
- using snake case seems standard in Ruby: https://namingconvention.org/ruby/